src/qiskit_sphinx_theme/theme/qiskit-sphinx-theme/custom_templates/sidebar_brand.html

```diff
@@ -36,4 +21,4 @@
   {%- endif %}
   {% endblock brand_content %}
 </a>
-{%- endif %}
+<span class="sidebar-ecosystem-link">This project is part of the <a href="https://qiskit.github.io/ecosystem/">Qiskit ecosystem</a></span>
```

Comment:
Does it look the same to use `<p>` here? Usually you use `<p>` when it's a newline, whereas `<span>` is for inline

```suggestion
<p class="sidebar-ecosystem-link">This project is part of the <a href="https://qiskit.github.io/ecosystem/">Qiskit ecosystem</a></p>
```

Reply:
Using `p` instead of `span` results in additional margins above and below (8 and 12 instead of 0 and 0. IMO `span` looks better because the vertical padding is already plenty. What do you suggest we do?

![image](https://github.com/Qiskit/qiskit_sphinx_theme/assets/4296166/7e26a276-70d7-491f-a55f-6d98027bccd2)

`span`:
![span](https://github.com/Qiskit/qiskit_sphinx_theme/assets/4296166/9f26220e-79d3-4371-8305-fed06e4a23fc)
`p`:
![p](https://github.com/Qiskit/qiskit_sphinx_theme/assets/4296166/f30613d4-1796-4404-8d95-bd06032c03d5)

Reply:
Oh I guess I can edit the padding since I defined it in a custom style.

Reply:
I changed it to `p` and got rid of the vertical padding